validator/validator.go

```diff
@@ -95,7 +95,7 @@ func (v *Validator) ValidateToken(ctx context.Context, tokenString string) (inte
 		return nil, fmt.Errorf("could not get token claims: %w", err)
 	}
 
-	userCtx := &UserContext{
+	userCtx := &ValidatedClaims{
```

Comment:
Now that we have only one validator, and it will be returning a pointer to `ValidatedClaims`, should the return type of `ValidateToken` still be `interface{}`?

Reply:
We could also update the name of the local variable `userCtx`

Reply:
The return type of `ValidateToken` should stay as it is to also allow users to implement their own validators if needed.